db/migrate/20160309144534_create_consoles.rb

```diff
@@ -0,0 +1,18 @@
+class CreateConsoles < ActiveRecord::Migration[5.0]
+  def change
+    create_table :consoles do |t|
+      t.string :protocol
+      t.string :host_name
+      t.integer :port
+      t.boolean :ssl
+      t.string :secret
```

Comment:
Does secret need to be encrypted in the database?  Is it like a password?  What exactly is it?